### ISSUE TITLE
Game crashes when resizing window MacOS

### ISSUE THREAD

**Issue by TeeJayBaker** [SheerSt/pokewilds#205]
Running on M1 Mac with Ventura and Java version 17.0.1

Game crashes when I attempt to manually resize the window
error code below

```
Save path: /users/tom
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
os = macOS
model = MacBookPro18,3
arch = ARM64 SoC: Firestorm + Icestorm
Scale: 3
Java version info
17.0.1
17
Graphics processor info
Apple
Apple M1 Pro
OpenGL info
2.1 Metal - 83
[Controllers] added manager for application, 1 managers active
Frame Buffers enabled: true
Setting else width = 160.000000, height = 144.000000
Setting else width = 162.666672, height = 144.666672
Setting else width = 163.333344, height = 144.666672
Setting else width = 164.666672, height = 146.000000
Setting else width = 166.666672, height = 147.333344
Setting else width = 175.333344, height = 155.333344
Setting else width = 178.000000, height = 158.000000
Setting else width = 182.000000, height = 162.666672
Setting else width = 194.666672, height = 180.666672
Setting else width = 196.666672, height = 184.000000
Setting else width = 200.666672, height = 190.000000
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x0000000182b75820, pid=11329, tid=5123
#
# JRE version: Java(TM) SE Runtime Environment (17.0.1+12) (build 17.0.1+12-LTS-39)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (17.0.1+12-LTS-39, mixed mode, sharing, tiered, compressed oops, compressed class ptrs, g1 gc, bsd-aarch64)
# Problematic frame:
# C  [libobjc.A.dylib+0x9820]  objc_msgSend+0x20
#
# No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /Users/tom/hs_err_pid11329.log
#
# If you would like to submit a bug report, please visit:
#   https://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
zsh: abort      java -jar /Users/tom/Downloads/pokewilds-v0.8.6-otherplatforms/pokewilds.jar
```

**Comment by fourlastor** [comment]
@TeeJayBaker this should be solved in the latest version (starting with 0.8.7), could you try updating the game and retrying?

**Comment by TeeJayBaker** [comment]
Thanks, Solved!
